File: tests/fixtures/fixture_java/src/main/java/com/app/services/AuthService.java
package com.app.services;

import com.app.models.User;

public class AuthService {
    public User authenticate(String email, String password) {
        if (email == null || password == null) return null;
        return new User("1", email, "Auth User");
    }

    public boolean validateToken(String token) {
        return token != null && token.length() > 10;
    }
}

File: tests/fixtures/fixture_csharp/Services/AuthService.cs
using App.Models;

namespace App.Services
{
    public class AuthService
    {
        public User Authenticate(string email, string password)
        {
            if (string.IsNullOrEmpty(email)) return null;
            return new User { Id = "1", Email = email, Name = "Auth User" };
        }

        public bool ValidateToken(string token)
        {
            return !string.IsNullOrEmpty(token) && token.Length > 10;
        }
    }
}
